src/app/views/river-detail/components/flow-tab/components/gage-summary.vue

```diff
@@ -15,9 +15,9 @@
           gage.adjusted_reach_class
         }}</span></div>
       <span class="gage-meta">
-        <span v-if="!gage.exclude && gage.delay_update">When updated this gage will wait {{ parseFloat(gage.delay_update/(3600)).toFixed(1) }} hours prior to reporting on the summary page. </span>
-        <span v-else-if="gage.exclude">When updated this gage will not report on the river summary page. </span>
-        <span v-else>When updated this gage will immediately report on the summary page. </span>
+        <span v-if="!gage.excluded && gage.delay_update">When updated this gage will wait {{ parseFloat(gage.delay_update/(3600)).toFixed(1) }} hours prior to reporting on the summary page. </span>
+        <span v-if="!gage.excluded && !gage.delay_update">When updated this gage will immediately report on the summary page. </span>
```

Comment:
any specific reason to stop using `else-if` here and instead just duplicate the conditions (`!gage.delay_update`)?

Reply:
In testing I could not get a gauge with delay update set to 0 and gauge.exclude set to true to display that it would not report on the river summary. It would report that it would immediately report on the river summary. By making this condition explicit if fixed the problem.  I agree it reads like it should have reported "not" but it definitely fell through to the final else until I added the explicit check.